### ISSUE TITLE
Poor documentation and samples

### ISSUE THREAD

**Issue by MehranDHN** [britishlibrary/TremendousIIIF#2]
It seems **TremendousIIIF** project is unique and the only one that exists in .Net framework. But the documentation is annoying. Beside that some of it's dependencies are deprecated which makes it very hard to follow and use.

**Comment by irv** [comment]
Hi, please see the api_3 branch for more complete documentation and dependencies

https://github.com/britishlibrary/TremendousIIIF/tree/api_3

**Comment by MehranDHN** [comment]
Thank you for your kind and immediate answer.
According to documentation I set the Location property in the ImageServer section to my local folder. But how can we sure that configuration is fine and it's ready to serve images?


![image](https://user-images.githubusercontent.com/9722881/103169855-bc632900-4854-11eb-8a1a-c9d68c04cdc1.png)

**Comment by irv** [comment]
try making a request for a filename that's available in your Location folder

e.g. GET /example.tiff/info.json

You can also set the values in your config for the health check identifier to be an image that you know exists and make a GET request to /_monitor/detail

**Comment by MehranDHN** [comment]
In both cases I've received 500 Server Error.
In my case  I copied an image named vishnu.jpg in local folder (C:\\JP2Cache\) that has been already configured in appsetings.json.
Sorry I really don't want to disturb you.  May be I should read more about IIIF Request/Response Model
![image](https://user-images.githubusercontent.com/9722881/103172872-e4aa5200-486b-11eb-8049-39e634ece9e3.png)

**Comment by irv** [comment]
Ah, only tiff is supported, or jpeg2000 if you have the commercial kakadu library. regular jpeg will not work
